AppCenterDataStorage/AppCenterDataStorage/MSDataStore.h

```diff
@@ -35,11 +35,19 @@ static NSString *const MSDataStoreUserDocumentsPartition = @"user";
 static NSString *const MSDataStoreAppDocumentsPartition = @"readonly";
 
 /**
- * Time to live constants
+ * Cache does not expire.
```

Comment:
It isn't a strong suggestion because I couldn't think of better doc for the constant. Maybe
```suggestion
 * No expiration on cache.
```